BibBuddy/app/src/main/java/de/bibbuddy/BookDao.java

```diff
@@ -485,4 +485,63 @@ public Long findShelfIdByBook(Long id) {
     return shelfId;
   }
 
+  /**
+   * Method that finds the shelf name of a book in the database.
+   *
+   * @param id id of the book
```

Comment:
```suggestion
   * @param id of the book
```
You usually do not use two similar words in param descriptions but rather continue after the param name with its declaration (you might want to check other classes outside our project to see some examples).